domain/model/character.go

```diff
@@ -1,5 +1,6 @@
 package model
 
+// Structure to hold the character data
```

Comment:
It would be great to make this more specific, as a good practice this comments help developing as this comment is the official doc of your struct, maybe adding this struct will contain information about pokemons...